<!-- GPS launch file -->

<launch>
		<node pkg="mr_gps" name="mr_gps" type="mr_gps" output="screen">
			<param name="ServerIP" value="10.115.253.233"/>  	
			<param name="ServerPort" value="21212"/>  
			<param name="requestMsg" value="Get position 5"/>  
			<param name="stateSub" value="mrGPS/State"/>  
			<param name="posePub" value="mrGPS/Pose"/>
			 
		</node>

		<node pkg="tf" type="static_transform_publisher" name="gps_tranform_broadcaster" args="-0.448 -1.27 0 -0.111 0 0 map gps_frame 100" />

		<node pkg="mr_gps" name="GPSLocalisation" type="GPSLocalisation" output="screen">
			<param name="odomTopic" value="/fmKnowledge/poses"/>  
			<param name="gpsPoseTopic" value="/mrGPS/Pose"/>  
			<param name="filterResetTopic" value="/initialpose"/>  
			<param name="gpsFrame" value="gps_frame"/> 
		</node>

</launch>
